terraform/modules/data-share-service/cloudwatch.tf

```
@@ -10,4 +10,28 @@ resource "aws_cloudwatch_log_group" "lb_sg_update" {
   retention_in_days = var.cloudwatch_retention_period
 
   kms_key_id = aws_kms_key.log_key.arn
+}
+
+resource "aws_cloudwatch_dashboard" "metrics_dashboard" {
+  dashboard_name = "metrics-dashboard"
+  dashboard_body = <<-EOF
+{
+  "widgets": [
+    {
+      "type": "metric",
+      "properties": {
+        "metrics": [
+          [
+            "gdxApp",
+            "API_CALLS.IngestedEvents.count"
+          ]
+        ],
+        "period": 300,
+        "region": ${var.region},
+        "title": "API calls"
+      }
+    }
+  ]
+}
+EOF
```

I don't think EOF actually will play nice here, my suggestion is use jsonencode

```suggestion
  dashboard_body = jsonencode(
{
  "widgets": [
    {
      "type": "metric",
      "properties": {
        "metrics": [
          [
            "gdxApp",
            "API_CALLS.IngestedEvents.count"
          ]
        ],
        "period": 300,
        "region": var.region,
        "title": "API calls"
      }
    }
  ]
})
```

---

Yeah this was the way the docs did it, but I can't see why else it's not working at that point, so will change it